app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.java

```diff
@@ -26,7 +26,12 @@
 		String[] leisures = {
 				"golf_course", "water_park", "miniature_golf", "dance",
 				"bowling_alley", "horse_riding", "sports_centre", "fitness_centre",
-				"amusement_arcade", "adult_gaming_centre", "tanning_salon" };
+				"amusement_arcade", "adult_gaming_centre", "tanning_salon",
+				"playground", "stadium", "recreation_ground", "sauna",
```

Comment:
leisure=playground would be spam as most are wheelchair=no
violate "Not an overwhelming percentage of elements have the same answer" requirement

Reply:
Well, I guess we're using different approaches on this tag on this. I think a playground is working fine for a wheelchair user if the entry is stepless, the width and smoothness of the paths are usable by a wheelchair user. That certain objects like slides are not usable by a wheelchair user (without help) is a completely different story.

But if there are no steps, reaching a playground, the ways are not too steep and bumpy, a wheelchair user can access the playground.

Reply:
"I think a playground is working fine for a wheelchair user if" - for start - is it documented somewhere on wiki?

Reply:
I tag only =yes when there is a special thing for wheelchairs there.

Reply:
@matkoniecz a combination of tags doesn't need to be documented somewhere to be used and make sense.

The question in playgrounds is really if the paths are passable by a wheelchair etc.

@HolgerJeromin this sounds pretty excluding.

Reply:
"The question in playgrounds is really if the paths are passable by a wheelchair etc." And I would rather expect that playground attractions are usable by somebody on a wheelchair.

Documentation is not necessary only for obvious things and in project as diverse as OSM nothing is obvious for everybody.

Reply:
> @HolgerJeromin this sounds pretty excluding.

How do you find such special stuff (when you need it) when everyone tags playgrounds access only?

Map the footways in high detail (and add wheelchair=yes) if you like.
But the sand pit itself is no fun with a wheelchair.